File: RestaurantProject.DataModel/Restaurant.cs
namespace RestaurantProject.DataModel
{
    public class Restaurant
    {
        public int Id { get; set; }

        public string Color { get; set; } = string.Empty;

        public string Address { get; set; } = string.Empty;

        public decimal Rent { get; set; }

        public Menu? Menu { get; set; }
    }
}

File: restaurant-client/src/types/restaurant.ts
export interface RestaurantDto {
  id: number
  color: string
  address: string
  rent: number
}

export interface CreateRestaurantDto {
  color: string
  address: string
  rent: number
}

export interface UpdateRestaurantDto {
  color: string
  address: string
  rent: number
}